warehouse/admin/templates/admin/banners/list.html

```diff
@@ -0,0 +1,51 @@
+{#
+ # Licensed under the Apache License, Version 2.0 (the "License");
+ # you may not use this file except in compliance with the License.
+ # You may obtain a copy of the License at
+ #
+ # http://www.apache.org/licenses/LICENSE-2.0
+ #
+ # Unless required by applicable law or agreed to in writing, software
+ # distributed under the License is distributed on an "AS IS" BASIS,
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+-#}
+{% extends "admin/base.html" %}
+
+{% block title %}Banners{% endblock %}
+
+{% block breadcrumb %}
+  <li class="active">Banners</li>
+{% endblock %}
+
+{% block content %}
+
+<div class="box box-primary">
+  {% if request.has_permission('admin') %}
+  <div class="box-header with-border">
+    <a class="btn btn-primary" href="#"><i class="fa fa-plus"></i> Create Banner</a>
+  </div>
+  {% endif %}
+  <div class="box-body table-responsive no-padding">
+    <table class="table table-hover">
+      <tr>
+        <th>Name</th>
+        <th>Is live?</th>
+        <th>Visibility period</th>
```

Comment:
Sure! Will do so!

Reply:
Changed in c8d0f415416e70417ec1369f3ad307a584f99010

I used a title instead of a more complex tooltip plugin because I didn't find anything similar in other html pages from admin.